docs/source/getting-started/what-is-jupyterhub.md

```diff
@@ -0,0 +1,465 @@
+components in the Jupyter ecosystem and everything else you read
+should make sense.
+
+
+
+## Just what is Jupyter?
+
+Before we get too far, let's remember what our end goal is.  A
+**Jupyter Notebook** is really nothing more than a Python(&) process
+which is getting commands from a web browser and displaying the output
+via that browser.  What the process actually sees can roughly like
+getting commands on standard input(&) and writing to standard
+output(&).  There is nothing intrinsically special about this process
+- it can do anything a normal Python process can do, and nothing more.
+The **Jupyter kernel** handles capturing output and converting things
+such as graphics to a form usable by the browser.
+
+Everything we explain below is building up to this, going through many
+different layers which give you many ways of customizing how this
+process runs.  But this process is not *too* special.
+
+
+
+## JupyterHub
+
+**JupyterHub** is the central piece that provides multi-user
+login capabilities. Despite this, the end user only briefly interacts with
+JupyterHub and most of the actual Jupyter session does not relate to
+the hub at all: the hub mainly handles authentication and creating (jargon: spawning) the
+single-user server.  In short, anything which is related to *starting*
+the user's workspace/environment is about JupyterHub, anything about
+*running* usually isn't.
+
+If you have problems connecting the authentication, spawning, and the
+proxy (explained below), the issues is usually with JupyterHub.  To
+debug, JupyterHub has extensive logs which get printed to its console
+and can be used to discover most problems.
+
+The main pieces of JupyterHub are:
+
+### Authenticator
+
+JupyterHub itself doesn't actually manage your users.  It has a
+database of users, but it is usually connected with some other system
+that manages the usernames and passwords.  When someone tries to log
+in to JupyteHub, it just asks the
+**authenticator**([basics](authenticators-users-basics),
+[reference](../reference/authenticators)) if the
+username/password is valid(&).  The authenticator returns a username(&),
+which is passed on to the spawner, which has to use it to start that
+user's environment.  The authenticator can also return user
+groups and admin status of users, so that JupyterHub can do some
+higher-level management.
+
+The following authenticators are included with JupyterHub:
+
+- **PAMAuthenticator** uses the standard Unix/Linux operating system
+  functions to check users.  Roughly, if someone already has access to
+  the machine (they can log in by ssh), they will be able to log in to
+  JupyterHub without any other setup.  Thus, JupyterHub fills the role
+  of a ssh server, but providing a web-browser based way to access the
+  machine.
+
+
+But those are fairly limited, and thus there are [plenty of others to
+choose
+from](https://github.com/jupyterhub/jupyterhub/wiki/Authenticators).
```

Comment:
let's keep markdown links all on the same line. In my opinion it's OK if documentation goes above typical code line length limits.

actually, I'd prefer that we stick with a "one sentence per line" approach. This makes it much easier to suggest changes to docs via the github UI